src/modules/Footer/components/Footer.js

```diff
@@ -25,6 +25,9 @@ const FooterBarUpper = styled.div`
 `
 
 const FooterLeft = styled.div`
+  @media ${MEDIA_QUERIES.MOBILE} {
```

Comment:
We don't need to define things like display: hidden all over again. Tailwind provides responsive classes out of the box. For example: <div className="hidden sm:inline">This div is hidden on the smallest screens and will be set as display: inline from the breakpoint 'sm' forward.</div>